can_tools/utils.py

```diff
@@ -86,3 +86,23 @@ def find_unknown_demographic_id(
 
     df_bad_combinations = df_bad[["age", "race", "ethnicity", "sex"]].drop_duplicates()
     return df_bad_combinations.to_csv(index=False, header=False)
+
+
+def find_duplicated_variable_entries(df: pd.DataFrame, has_location=False):
```

Comment:
this detects any duplicated variables. Sometimes scrapers have multiple entries for the same variable (either due to an issue with our scraping or with the dashboard). This helper function highlights those rows (if any exist)